StreamChatClient_v3/APIClient/Endpoints/Payloads/ChannelCreatePayload.swift

```diff
@@ -0,0 +1,58 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+public struct ChannelCreatePayload<ExtraData: ExtraDataTypes>: Encodable {
+    let cid: ChannelId
+    let name: String?
+    let imageURL: URL?
+    let team: String?
+    let members: Set<UserId>
+    let invites: Set<UserId>
+    let extraData: ExtraData.Channel?
+
+    init(
+        cid: ChannelId,
+        name: String?,
+        imageURL: URL?,
+        team: String?,
+        members: Set<UserId>,
+        invites: Set<UserId>,
+        extraData: ExtraData.Channel?
+    ) {
+        self.cid = cid
+        self.name = name
+        self.imageURL = imageURL
+        self.team = team
+        self.members = members
+        self.invites = invites
+        self.extraData = extraData
+    }
+
+    private enum EncodingKeys: String, CodingKey {
+        case name, imageURL, team, members, invites, extraData
+    }
```

Comment:
We can reuse the keys from `ChannelDetailPayload`. They should/must be the same as these.

Reply:
`extraData` don't have a special key. They are encoded directly in the body of the object. So, in the `encode` method, you just need to do `try extraData.encode(to: encoder)`